gnucash/gnome/gnc-budget-view.c

```diff
@@ -984,6 +985,18 @@ budget_col_source(Account *account, GtkTreeViewColumn *col,
                          NULL);
         }
     }
+
+    note = gnc_budget_get_account_period_note(budget, account, period_num);
+
+    if (note == NULL)
+    {
+        g_object_set(cell, "background", NULL, NULL);
+    }
+    else
+    {
+        g_object_set(cell, "background", "beige", NULL);
```

Comment:
Don't hardcode colours. This doesn't work well with custom themes. Instead query the theme for a proper colour based on the function of the color.

If you can't find a suitable colour in the theme, you should provide it in our own css files so at least a user can override it. This will probably mean you'll have to set a custom css class for each cell which is different for cells with note or without and then have a css snippet defining a background color based on this css class.
Note our css files cater for both light or dark themes.